cts/cli/regression.tools.exp

```diff
@@ -23,6 +23,481 @@ A new shadow instance was created. To begin using it, enter the following into y
 </cib>
 =#=#=#= End test: Validate CIB - OK (0) =#=#=#=
 * Passed: cibadmin       - Validate CIB
+=#=#=#= Begin test: List all available cluster options =#=#=#=
+Pacemaker cluster options
+
+Also known as properties, these are options that affect behavior across the entire cluster. They are configured within cluster_property_set elements inside the crm_config subsection of the CIB configuration section.
+
+  * dc-version: Pacemaker version on cluster node elected Designated Controller (DC)
+    * Includes a hash which identifies the exact revision the code was built from. Used for diagnostic purposes.
+    * Possible values (generated by Pacemaker): string (default: )
+
+  * cluster-infrastructure: The messaging layer on which Pacemaker is currently running
+    * Used for informational and diagnostic purposes.
+    * Possible values (generated by Pacemaker): string (default: )
```

Comment:
@tomjelinek @gao-yan , here's what the text output of the new crm_attribute option would look like. If you're not concerned with structured data, you could just call that and be done with it. (The regression tests strip the default values because they may depend on configure options. The actual output would show the defaults.)